readthedocs/config/tests/test_config.py

```diff
@@ -691,7 +665,7 @@ def test_use_conda_respects_config():
         get_env_config(),
     )
     build.validate()
-    assert build.use_conda is True
+    assert build.conda is not None
```

Comment:
Maybe it worth to have a better check here. Using `isinstance(dict)` or something like that plus something else (check for specific keyword). `not None` could be any thing.